packs/feats/deep-roots.json

```
@@ -24,7 +24,28 @@
             "remaster": false,
             "title": "Pathfinder Dark Archive"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "item:tag:deep-roots",
+                "placement": "spellcasting",
+                "toggleable": true
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:tag:deep-roots"
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.Psychic.DeepRoots.Description"
+                    }
+                ]
+            }
```

```suggestion
                "key": "RollOption",
                "option": "imposes:controlled",
                "placement": "spellcasting",
                "toggleable": true
            },
            {
                "itemType": "spell",
                "key": "ItemAlteration",
                "mode": "add",
                "predicate": [
                    "imposes:controlled"
                ],
                "property": "description",
                "value": [
                    {
                        "text": "PF2E.SpecificRule.Psychic.DeepRoots.Description"
                    }
                ]
            }
```
For the other one we can auto tag the spells grants with the ancestral spell tag. This this one we don't need tags, and instead go to the normal way we mark these kind of roll options, though it's likely going to be a while before it's truly automated.